add tips for advanced debugging (#752)
conda/constructor: docs/source/cli-options.md

```

:::{admonition} Extra verbosity
:class: tip

Get more verbose outputs by running the installer with `-x` and/or `CONDA_VERBOSITY=3`:

```bash
$ bash -x my_installer.sh
# or with verbose conda:
$ CONDA_VERBOSITY=3 bash -x my_installer.sh
```
:::

## Windows installers

Windows installers have the following CLI options available:

- `/InstallationType=[JustMe|AllUsers]`: This flag sets the installation type. The default is
  `JustMe`. `AllUsers` might require elevated privileges.
- `/AddToPath=[0|1]`: Whether to add the installation directory to the `PATH` environment
  variable. The default is `0`. This is NOT recommended.
- `/CheckPathLength=[0|1]`: Check whether the installation path is too long (>46
  characters). The default depends on how the installer was created.
- `/KeepPkgCache=[0|1]`: Whether to keep the package cache or not. Defaults to `1`.
- `/NoRegistry=[0|1]`: Whether to prevent registry modification or not. Defaults to `0`.
- `/NoScripts=[0|1]`: If set to `1`, the installer will not run any post-install scripts. Defaults
  to `0`.
- `/NoShortcuts=[0|1]`: If set to `1`, the installer will not create any shortcuts. Defaults to
  `0`.

> - `/RegisterPython=[0|1]`: Whether toregister Python as default in the Windows registry. Defaults

Reviewer: ```suggestion
- `/RegisterPython=[0|1]`: Whether to register Python as default in the Windows registry. Defaults
```

> cmd.exe /c start /wait "C:\Program Files\my_app\uninstaller.exe" /S _?=C:\Program Files\my_app
```

:::{admonition} EXE installers with file logging
:class: tip

Windows installers do not have a verbose mode. By default, the graphical logs are only available in the "progress bar" dialog, by clicking on "Show details". This text box is right-clickable, which will allow you to copy the contents to the clipboard (and then paste them in a text file, presumably).

> There's a way of building EXE installers that do write logs to a file. For this, you need a special `nsis` package configured to do so:

Reviewer: ```suggestion
There's a way of building EXE installers that can write logs to a file; for this, you need a special `nsis` package configured to do so:
```

```batch
> conda install "nsis=*=*log*"
```

> Then, you can invoke `constructor` normally, but setting a special environment variable:

Reviewer: ```suggestion
Then, you can invoke `constructor` normally after setting a special environment variable:
```
